src/convex/workspaces.ts

```
@@ -77,39 +75,28 @@ export const createWorkspace = mutation({
 
-		const existing = await ctx.db
-			.query('workspaces')
-			.withIndex('by_slug', (q) => q.eq('slug', slug))
-			.unique();
-		if (existing) {
-			throw new ConvexError({
-				code: 'already_exists',
-				message: 'Workspace slug already in use'
-			});
+		const handleBase = workspaceHandleBase(name);
+		let slug = handleBase;
```

ESLint

[error] The value assigned to 'slug' is not used in subsequent statements.

(no-useless-assignment)